plugins/szero.yaml

```
@@ -0,0 +1,116 @@
+
+    - selector:
+        matchLabels:
+          os: linux
+          arch: "386"
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_linux_386.tar.gz
+      sha256: d3fa74ab85af8920fe3a2f7cae3b778870e219d2694850eb0d3173bbe3df31bf
+      files:
+        - from: "./szero"
+          to: "kubectl-szero"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero"
+
+    - selector:
+        matchLabels:
+          os: linux
+          arch: "arm64"
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_linux_arm64.tar.gz
+      sha256: 28ca17e0cf87f84315db859daf6132d37671de347c6f3a1070b18f7913c86108
+      files:
+        - from: "./szero"
+          to: "kubectl-szero"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero"
+
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_darwin_amd64.tar.gz
+      sha256: 6f7f39e411c05e610841d3452b9a71c6aa7f3338ec587d06c15c9ebef8387181
+      files:
+        - from: "./szero"
+          to: "kubectl-szero"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero"
+
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: arm64
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_darwin_arm64.tar.gz
+      sha256: 7c87234ec25b930c806cf7148922a304b0ef5bf2a3c8c0109b932f609f6848ee
+      files:
+        - from: "./szero"
+          to: "kubectl-szero"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero"
+
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_windows_amd64.zip
+      sha256: 94b12498ce04bea9236a5a99e2f109712fcbbbf52e5e277f64643434a1fa51fd
+      files:
+        - from: "./szero.exe"
+          to: "kubectl-szero.exe"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero.exe"
+
+    - selector:
+        matchLabels:
+          os: windows
+          arch: "386"
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_windows_386.zip
+      sha256: d2f9591a2597aa6b95a01c23dd99260c4d4ed7e9026e0d8c946d704c7f3f5162
+      files:
+        - from: "./szero.exe"
+          to: "kubectl-szero.exe"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero.exe"
+
+    - selector:
+        matchLabels:
+          os: windows
+          arch: arm64
+      uri: https://github.com/jadolg/szero/releases/download/v1.11.0/szero_windows_arm64.zip
+      sha256: 7769bca78775337c675745b0e397e15bdd7b609f8e997b691a109cff547f60d1
+      files:
+        - from: "./szero.exe"
+          to: "kubectl-szero.exe"
+        - from: LICENSE
+          to: "."
+      bin: "kubectl-szero.exe"
+
+  shortDescription: Temporarily scale down/up all deployments, statefulsets, and daemonsets in a namespace
```

This should be max 55 chars otherwise it's always clipped.
```suggestion
  shortDescription: Scale all workloads in a namespace to 0 and back
```